database/migrations/sqls/20230711160946-trees-denormalized-up.sql

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE trees_denormalized (
+    id int NOT NULL PRIMARY KEY,
+    organizations jsonb NOT NULL,
```

Comment:
@Kpoke the `organizations` should be Array here, we tested the type, Array performs better than jsonb.